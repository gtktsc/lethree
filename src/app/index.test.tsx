import { render } from 'enzyme';
import App from './index';

describe('init tests', () => {
    test('make sure tests run', () => {
        const app = render(App);
        expect(app).toMatchSnapshot();
    });
});
