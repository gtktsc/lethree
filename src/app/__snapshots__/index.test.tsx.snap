// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init tests make sure tests run 1`] = `
initialize {
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
